source/gameengine/Converter/BL_BlenderDataConversion.cpp

```diff
@@ -1858,25 +1857,16 @@ void BL_ConvertBlenderObjects(struct Main* maggie,
 						continue;
 					}
 
-					if (tex->IsCubeMap()) {
-						KX_GameObject *viewpoint = gameobj;
-
-						if (env->object) {
-							KX_GameObject *obj = converter->FindGameObject(env->object);
-							if (obj) {
-								viewpoint = obj;
-							}
+					KX_GameObject *viewpoint = gameobj;
+					if (env->object) {
+						KX_GameObject *obj = converter->FindGameObject(env->object);
+						if (obj) {
+							viewpoint = obj;
 						}
-
-						kxscene->GetCubeMapManager()->AddCubeMap(tex, viewpoint);
-					}
-					else {
-// 						EnvMap *env = tex->GetTex()->env;
-						KX_GameObject *viewpoint = gameobj;
-						// Prevent two sided rendering on mirror's material to avoid artifacts
-						polymat->SetRasMode(0);
```

Comment:
iirc this was used to prevent two sided rendering on planar reflections because I had artifacts with backface culling enabled (because of switch beetween frontface/backface during planar rendering). I have to check if with your implementation the problems are gone.